.dependabot/config.yml

```diff
@@ -5,3 +5,15 @@ update_configs:
   - package_manager: "terraform"
     directory: "/"
     update_schedule: "daily"
+
```

Comment:
we should probably add `go.dep` and `go.sum` here too
```suggestion

  - package_manager: "go:dep"
    directory: "/tests"
    update_schedule: "daily"

  - package_manager: "go:modules"
    directory: "/tests"
    update_schedule: "daily"
```

Reply:
Fixed

Reply:
We're not actually using `go:dep`, that's the one we replaced with go modules. Doesn't hurt to be here, just fyi.

Reply:
I only added the configuration for `go:modules`